src/test/integration/web_interface/rest/test_rest_missing_analyses.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from storage.db_interface_backend import BackEndDbInterface
+from test.common_helper import create_test_file_object, create_test_firmware
+from test.integration.web_interface.rest.base import RestTestBase
+
+
+class TestRestMissingAnalyses(RestTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.db_backend = BackEndDbInterface(config=cls.config)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.db_backend.shutdown()
+        super().tearDownClass()
+
+    def test_rest_get_missing_files(self):
+        test_fw_1 = create_test_firmware()
+        missing_uid = 'uid1234'
+        test_fw_1.files_included.add(missing_uid)
+        self.db_backend.add_firmware(test_fw_1)
+
+        response = json.loads(self.test_client.get('/rest/missing_analyses', follow_redirects=True).data.decode())
+        assert 'missing_files' in response
+        assert test_fw_1.uid in response['missing_files']
+        assert missing_uid in response['missing_files'][test_fw_1.uid]
+        assert response.get('missing_analyses') == {}
+
+    def test_rest_get_missing_analyses(self):
```

Comment:
Test is sort of confusing. But I don't see a better way given the task at hand.

Reply:
added comment that explains how the test works